lib/backend/k8s/resources/customresource.go

```diff
@@ -301,26 +295,9 @@ func (c *customK8sResourceClient) List(list model.ListInterface) ([]*model.KVPai
 	return kvps, reslOut.GetListMeta().GetResourceVersion(), nil
 }
 
-// EnsureInitialized performs client initialization required for a specific
-// Custom K8s Resource type.
+// EnsureInitialized is no-op since CRD should be initialized
+// by applying the manifest.
```

Comment:
what manifest?  Someone reading the code might not know.

I'd just say "initialized in advance"